solr/solr-ref-guide/modules/deployment-guide/pages/solr-on-hdfs.adoc

```diff
@@ -246,6 +246,17 @@ s|Required |Default: none
 +
 The Kerberos principal that Solr should use to authenticate to secure Hadoop; the format of a typical Kerberos V5 principal is: `primary/instance@realm`.
 
+== Shard splitting settings
```

Comment:
The *only* point of putting this documentation here, on the Hdfs page, would be **if** we would recommend the user to do something about this setting if they are using Hdfs.  But you flipped this setting off for HdfsDirectory and so I think it's now misplaced.  Instead, I'd specify this over in a shard split page.

And when yhou do, ensure it's clear that this is a *system property*; not a parameter or something.